### PR TITLE
update composer.json for yii-gii new version and newer version

### PR DESCRIPTION
**Pull request by liasica**
fix error:
```
Problem 1
    - Can only install one of: yiisoft/yii2-gii[2.0.x-dev, 2.1.0].
    - Can only install one of: yiisoft/yii2-gii[2.0.x-dev, 2.1.0].
    - schmunk42/yii2-giiant dev-master requires yiisoft/yii2-gii ~2.0.6 -> satisfiable by yiisoft/yii2-gii[2.0.x-dev].
    - Installation request for schmunk42/yii2-giiant dev-master -> satisfiable by schmunk42/yii2-giiant[dev-master].
    - Installation request for yiisoft/yii2-gii ~2.1.0 -> satisfiable by yiisoft/yii2-gii[2.1.0].
```

### CODE REVIEW

**Comment by liasica** [comment]
close my pr: crashed for yii-gii 2.1.0 sorry.

**Comment by liasica** [comment]
tested works good :)

File: composer.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
         "dmstr/yii2-db": "*",
         "friendsofphp/php-cs-fixer": "1.* || 2.*",
         "yiisoft/yii2": "~2.0.13",
-        "yiisoft/yii2-gii": "~2.0.6"
+        "yiisoft/yii2-gii": "2.*"
```
Comment (schmunk42) [inline composer.json]: This should be more like: `~2.0.6 || ~2.1.0`

**Comment by schmunk42** [comment]
Related: https://github.com/schmunk42/yii2-giiant/issues/251

File: composer.json (hunk 1 above)
Comment (liasica) [reply composer.json]: yeah, you are right.

**Comment by schmunk42** [comment]
Thank you!

**Pull request merged**